TM1py/Services/HierarchyService.py

```diff
@@ -428,6 +428,7 @@ def update_or_create_hierarchy_from_dataframe(
             verify_edges: bool = True,
             element_type_column: str = 'ElementType',
             unwind: bool = False,
+            unwind_consol: str = '',
```

Comment:
The scalable approach would be to create an iterator (providing one or more consolidations).

Reply:
Thanks for the feedback! Yes, that would definitely help to manage the alternate rollups.

> The scalable approach would be to create an iterator (providing one or more consolidations).